https.go

```diff
@@ -246,7 +246,11 @@ func (proxy *ProxyHttpServer) handleHttps(w http.ResponseWriter, r *http.Request
 						return
 					}
 					if err != nil {
-						ctx.Warnf("Illegal URL %s", "https://"+r.Host+req.URL.Path)
+						if req != nil {
```

Comment:
Drive by review: Should this be `if req.URL != nil`? If `req` is `nil`, then I'd expect `isWebSocketRequest` above to panic.

Reply:
Yes, good catch. I've changed to `if req.URL != nil` instead.